test/e2e/nemo-dependencies/values.yaml

```diff
@@ -4,7 +4,7 @@ install:
   embedding: no
   ranking: no
   entity_store: no
-  evaluator: no
+  evaluator: yes
```

Comment:
lets leave this off by default and users enable necessary components.

Reply:
Yes. Forgot to change it back after testing. Fixed it